application/src/test/java/org/thingsboard/server/system/BaseHttpDeviceApiTest.java

```diff
@@ -74,6 +79,26 @@ public void testGetAttributes() throws Exception {
         doGetAsync("/api/v1/" + deviceCredentials.getCredentialsId() + "/attributes?clientKeys=keyA,keyB,keyC").andExpect(status().isOk());
     }
 
+    @Test
+    public void testReplyToCommandWithLargeResponse() throws Exception {
+        String errorResponse = doPostAsync("/api/v1/" + deviceCredentials.getCredentialsId() + "/rpc/5",
+                JacksonUtil.toString(createRpcResponsePayload(101)),
+                String.class,
+                status().is5xxServerError());
+        assertThat(errorResponse).isEqualTo("Max message size exceeded for device [" + device.getId() + "]");
+
+        doPostAsync("/api/v1/" + deviceCredentials.getCredentialsId() + "/rpc/5",
+                JacksonUtil.toString(createRpcResponsePayload(10)),
```

Comment:
I would like to see the corner case. like Maximum allowed payload ok. And failed `max + 1` or any expected overflow